src/model/generate_model.jl

```diff
@@ -88,32 +88,74 @@ function generate_model(setup::Dict,inputs::Dict,OPTIMIZER::MOI.OptimizerWithAtt
 
 	# Initialize Power Balance Expression
 	# Expression for "baseline" power balance constraint
-	@expression(EP, ePowerBalance[t=1:T, z=1:Z], 0)
+	# @expression(EP, ePowerBalance[t=1:T, z=1:Z], 0)
+	temp = Matrix{AffExpr}(undef, T, Z)
+	for t=1:T
+		for z=1:Z
+			temp[t,z] = 0.0
+		end
+	end
+	# @expression(EP, ePowerBalance[t=1:T, z=1:Z], temp[t, z])
+	EP[:ePowerBalance] = temp
 
 	# Initialize Objective Function Expression
-	@expression(EP, eObj, 0)
+	# @expression(EP, eObj, 0)
+	# createemptyexpression(EP, :eObj, 1)
+	temp = AffExpr(0.0)
+	EP[:eObj] = temp
+
 
 
 	#@expression(EP, :eCO2Cap[cap=1:inputs["NCO2Cap"]], 0)
-	@expression(EP, eGenerationByZone[z=1:Z, t=1:T], 0)
+	# @expression(EP, eGenerationByZone[z=1:Z, t=1:T], 0)
+	createemptyexpression(EP, :eGenerationByZone, Z, T)
 	# Initialize Capacity Reserve Margin Expression
 	if setup["CapacityReserveMargin"] > 0
-		@expression(EP, eCapResMarBalance[res=1:inputs["NCapacityReserveMargin"], t=1:T], 0)
+		temp = Matrix{AffExpr}(undef, inputs["NCapacityReserveMargin"], T)
+		for t=1:T
+			for x=1:inputs["NCapacityReserveMargin"]
+				temp[x, t] = 0.0
+			end
+		end
```

Comment:
again, I think this can be replaced by one call to `zeros(AffExp, inputs["N..."], T)`